plugin/fzf.vim

```diff
@@ -537,6 +549,15 @@ function! s:execute(dict, command, use_height, temps) abort
       call jobstart(cmd, fzf)
       return []
     endif
+  elseif has('win32unix')
+    if $TERM ==# 'cygwin'
```

Comment:
Sure.
I intended the vim plugin to be compatible with the current binaries for Windows but cygwin is not officially supported yet.